ci-operator/step-registry/vsphere/deprovision/usertags/vsphere-deprovision-usertags-commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+tag_category_name="${CLUSTER_NAME}-usertags-category"
+
+printf '%s' "${USER_TAGS:-}" | while read -r tag
+do
+    if [[ -n "$(govc tags.ls | grep ${tag})" ]];then
```

Comment:
`govc tags.ls` will print out all the tags, we should limit it to the ones only under `tag_category_name`, right?

Reply:
And we should firstly to ensure `tag_category_name` exits, then start to loop the tags under the category, right?